frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunDetails.tsx

```diff
@@ -121,8 +122,8 @@ const PipelineRunDetails: PipelineCoreDetailsPageComponent = ({ breadcrumbPath,
                   breadcrumb={
                     <Breadcrumb>
                       {breadcrumbPath}
-                      <BreadcrumbItem isActive>
-                        {error ? 'Run details' : run?.name ?? 'Loading...'}
+                      <BreadcrumbItem isActive style={{ width: 300 }}>
+                        <Truncate content={error ? 'Run details' : run?.name ?? 'Loading...'} />
```

Comment:
Here I have added width, beacuse of the long run name,the breadcrumitem is displaying in the next line. I need some suggestions here, whether to keep it as it is without applying width or any other ways to solve this.

Reply:
Best to use `maxWidth` instead of `width`.

I don't know if there's really anything else you can do here to limit the width as the `Breadcrumb` component is meant to wrap.